renovate.json5

```diff
@@ -3,6 +3,21 @@
     "apollo-open-source"
   ],
   "dependencyDashboard": true,
+  // The "circleci" manager is intentionally disabled right now (e.g., not
+  // included in this list).  While we do benefit from its updating of "Orb"
+  // versions, the CircleCI manager also attempts to update Docker images using
+  // its "Docker" datasource.  This is really handy, in theory, but we have
+  // intentionally varying major Node.js docker image identifiers in our
+  // CircleCI configuration to test on each major Node.js platform.  Enabling
+  // the "circleci" manager would cause all of these to update to the latest
```

Comment:
I'm not trying to verify whether this is true or not (I kinda hope it's actually not hard to tell it not to take major version bumps), happy to move forward with circleci disabled for now even if there's a chance you're being overly conservative.

Reply:
I suspect it's easy to tell it to not take major version bumps for this file, yes.